Documentation/articles/content/why_content_pipeline.md

```diff
@@ -0,0 +1,69 @@
+# Why use the Content Pipeline
+
+The MonoGame team has been putting a lot of effort into a cross-platform content pipeline, but would you use the **Content Pipeline** when MonoGame also supports loading assets natively, like .png, .mp3, .wav? Well, it all boils down to a couple of words, performance and efficiency.
+
+## Textures
+
+Most texture formats (like PNGs) are generally not optimized for games.
+
+For example, most GPUs have specific hardware supported compressed formats for textures. PNGs or JPEGs just get uncompressed when passed to the GPU. By using the hardware compression you often get 4x to 8x more textures space for your game as well as faster load times and smaller packages.
+
+If we have a 256×256 32 bit .png texture that we are using in our game, why should a developer bother with compiling to .xnb's, they just want to use the texture as a raw .png file. On disk .png is very impressive in its size, that image probably only takes up 2-5 kb on disk, keeping your application package size down. Great!
```

Comment:
> If we have a 256×256 32 bit .png texture that we are using in our game, why should a developer bother with compiling to .xnb's, they just want to use the texture as a raw .png file.

This sentence makes no sense.  I would cut everything after the first comma... the rest is unnecessary info.

Reply:
For Example, [redacted]
:D

Reply:
Updated section and removed hyperbole